src/unity/python/turicreate/toolkits/drawing_recognition/util/_visualization.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2019 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+import turicreate as _tc
+from turicreate import extensions as _extensions
+
+def draw_strokes(stroke_based_drawings):
+    if type(stroke_based_drawings) != _tc.SArray:
+        raise _ToolkitError("Input to draw_strokes must be of type " 
+            + "turicreate.SArray")
+    is_stroke_input = (stroke_based_drawings.dtype != _tc.Image)
+    if is_stroke_input:
+        sf = _tc.SFrame({"drawings": stroke_based_drawings})
+        sf_with_drawings = _extensions._drawing_recognition_prepare_data(
+            sf, "drawings")
+        return sf_with_drawings["drawings"]
+    else:
+        # User passed in drawings that were already drawings
```

Comment:
Why do we want/need to support this?

Reply:
In the event that the user passes in an SFrame with drawings to the `draw_strokes` function, I thought it was better to silently return the same SFrame instead of throwing an error. Although now that I think about it, passing in an SFrame with drawings is a user error and we should probably throw an error for it. Thoughts?

Reply:
Yeah, you already have it as an error to pass an SFrame. If anything, you might consider accepting either an SArray of lists of dictionaries, or just one list of dictionaries.